tests/unit/src/commands/initialize/initDeployment.ts

```diff
@@ -0,0 +1,211 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import loadModule, { cleanupModuleMocks } from '../../../../_support/loadModule';
+import { spy, stub, SinonSpy, SinonStub } from 'sinon';
+
+let initDeployment: any;
+let Travis: any;
+let GitHub: any;
+let TravisSpy: SinonSpy;
+let githubAuthStub: SinonStub;
+let keyFileStub: SinonStub;
+let encryptedKeyFileStub: SinonStub;
+let createDeployKeyStub: SinonStub;
+let findStub: SinonStub;
+let existsSyncStub: SinonStub;
+let readFileSyncStub: SinonStub;
+let isAuthorizedStub: SinonStub;
+let travisCreateAuthorizationStub: SinonStub;
+let travisDeleteAuthorizationStub: SinonStub;
+let fetchRepositoryStub: SinonStub;
+let listEnvironmentVariablesStub: SinonStub;
+let setEnvironmentVariablesStub: SinonStub;
+let createKeyStub: SinonStub;
+let deleteKeyStub: SinonStub;
+
+registerSuite({
+	name: 'commands/initialize/initDeployment',
+
+	before() {
+		githubAuthStub = stub();
```

Comment:
Since these are only ever set once, and are reset each test run, they can be initialized above where they're declared and `let` can change to `const`.